### PR TITLE
Pract.02

### PR DESCRIPTION
**Pull request by Stanislava50**
Tasks Prac2
Stanislava Zhelyazkova
4MI0600199

### CODE REVIEW

**Comment by GeorgiTerziev02** [comment]
For pract 02 the new rules for the PRs will not be considered. Do not close this pull request.

File: SI/Sem.02/Pract.02/Task01.cpp [hunk 1]
```diff
@@ -1,16 +1,22 @@
-#include <iostream>
-
-int main() {
-	int year = 0;
+// task01.cpp : This file contains the 'main' function. Program execution begins and ends there.
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/Task01.cpp:1]: For the first 3 tasks you can see example solutions that are already merged.

File: SI/Sem.02/Pract.02/Task02.cpp [hunk 2]
```diff
@@ -1,33 +1,50 @@
...
+	{
+		cout << "This symbol is from the Latin alphabet."<< endl;
+	}
+	else
+	{ 
+		cout << "This symbol is not from the Latin alphabet" << endl; 
+	}
+	switch(symbol)
+	{
+	case'A':cout << "vowel";
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/Task02.cpp:22]: You can check the solution that is in Pract.02/task02.cpp.
The problem here is we are repeating a lot of our code unnecessary.

File: SI/Sem.02/Pract.02/Task03.cpp [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 
-	if (number1 == number2) {
-		std::cout << "Both numbers are equal." << std::endl;
-		return 0;
-	}
+#include <iostream>
+using namespace std;
+int main()
+{
+	//�������� ��������, ����� ������ ��� �������� ����� � ����� ��� � �� ������.
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/Task03.cpp:8]: You are having problems with the encoding you use on the comments.
Are you writing comments in Cyrillic?

File: SI/Sem.02/Pract.02/task04.cpp [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+	}
+	else if(result>=60)
+	{
+		cout << "The mark is:4";
+	}
+	else if(result>=40)
+	{
+		cout << "The mark is:3";
+	}
+	else if (result<40)
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/task04.cpp:38]: Can our percentages be negative?
Shouldn't we cover this case?

File: SI/Sem.02/Pract.02/task04.cpp [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+	cin >> result;
+	if (result >= 90)
+	{
+		cout << " The mark is: 6+";
+	}
+	else if (result >= 80)
+	{
+		cout << "The mark is:6";
+	}
+	else if(result>=70)
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/task04.cpp:26]: keep consistency and good practices.
Tabulations/Spaces between the operations and variables.
E.g: else if(result >= 70)

File: SI/Sem.02/Pract.02/task06.cpp [hunk 6]
```diff
@@ -0,0 +1,22 @@
+// ConsoleApplication4.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+	int a, b, c;
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/task06.cpp:9]: Re-visit this task.
- Shouldn't the discriminant be a separate variable?
- Do we always have roots?
- Do we always have a quadratic equation?

File: SI/Sem.02/Pract.02/task08.cpp [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int numeral;
+    cout << "Arabic numeral:";
+    cin >> numeral;
+    if (numeral == 1)
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/task08.cpp:12]: Re-write this task with a switch case.

File: SI/Sem.02/Pract.02/task07.cpp [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+using namespace std;
+
+int main()
+//�� �� ������ ��������, ����� ���� �� ��������� ������ �, ��� � ����� �����, ������� ����������� ������, ��� � ������ �����,
+//������� ����������� �����, � ��� �� � �����, �� ������ ��������� �� ��������� ������.
+{
+	char symbol;
+	cout << "Enter a symbol: ";
+	cin >> symbol;
+	if ('a' <= symbol <= 'z')
```
Comment (Justsvetoslavov) [inline SI/Sem.02/Pract.02/task07.cpp:14]: Re-visit this task.
- 'a' <= symbol <= 'z' does this work?
- if you have 2 conditions -> logical operator &&
- Check operator += and -=

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
Currently reviewed until task 8(included).
- Fix comments
- add spaces between operations as mentioned in some of the tasks.

File: SI/Sem.02/Pract.02/task10.cpp [hunk 9]
```diff
@@ -0,0 +1,43 @@
+// ConsoleApplication10.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+
+int main()
+//�������� ����� ����������, ����� �������� ���������� + , -, *, / �� ��� ������ �����.
```
Comment (dejansvetoslavov) [inline SI/Sem.02/Pract.02/task10.cpp:8]: We don't speak this language 😆

File: SI/Sem.02/Pract.02/task10.cpp [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+{
+    int a, b;
+    char sign;
+    cout << "Enter the first number:";
+    cin >> a ;
+    cout << "Enter operator sign:";
+    cin >> sign;
+    cout << "Enter the second number:";
+    cin >> b;
+    switch(sign)
```
Comment (dejansvetoslavov) [inline SI/Sem.02/Pract.02/task10.cpp:19]: Fix formatting.

File: SI/Sem.02/Pract.02/task11.cpp [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+using namespace std;
+
+int main()
+//�� �� ������� �� ��������� 3 ����� � �� �� ������ ��� - �������� �����, ����� ���� �� �� ������,
+//��� ���� ����� �� �������, �������� � �������.
+{
+	int a, b, c;
+	cout << "Enter 3 digits:";
+	cin >> a >> b >> c;
+	if(a>b && a>>c && b>c)
```
Comment (dejansvetoslavov) [inline SI/Sem.02/Pract.02/task11.cpp:14]: This is wrong

**Review by dejansvetoslavov — CHANGES_REQUESTED** [review]

**Comment by Justsvetoslavov** [comment]
Adding label duplicate.
Closing PR due to single commit and unable to merge.

**Pull request closed without merging**